quadrat/sass/blocks/_post-title.scss

```diff
@@ -0,0 +1,9 @@
+.wp-block-post-title {
+	margin-bottom: calc( var(--wp--custom--gap--vertical) * 3.8 );
```

Comment:
When there is a featured image, this causes more space between the title and featured image:

Trunk | This PR
------ | -----
<img width="862" alt="Screen Shot 2021-09-30 at 2 22 13 PM" src="https://user-images.githubusercontent.com/5375500/135510168-36edff47-93fb-4a6f-9894-64743106ad78.png"> | <img width="867" alt="Screen Shot 2021-09-30 at 2 22 22 PM" src="https://user-images.githubusercontent.com/5375500/135510182-318fe935-9c9a-4f45-9257-167a770b9d7f.png">

I don't know if that's desirable cc @kjellr @beafialho

Reply:
That's not desirable. 😄 We should have equal spacing above and below the title. (Ideally the spacing would below the title would match the spacing that's currently above the title).